src/WebJobs.Script.WebHost/Diagnostics/LinuxContainerEventGenerator.cs

```diff
@@ -68,6 +69,7 @@ public override void LogFunctionTraceEvent(LogLevel level, string subscriptionId
             string eventTimestamp = DateTime.UtcNow.ToString(EventTimestampFormat);
             string hostVersion = ScriptHost.Version;
             FunctionsSystemLogsEventSource.Instance.SetActivityId(activityId);
+            details = details.Substring(0, MaxDetailsLength);
```

Comment:
I believe we would need to account for the case where the actual length is less than `MaxDetailsLength`. Adding a conditional check should be sufficient.